services/apps/alcs/src/alcs/application/application-submission/application-submission.service.ts

```diff
@@ -17,6 +18,8 @@ export class ApplicationSubmissionService {
     private applicationSubmissionRepository: Repository<ApplicationSubmission>,
     @InjectRepository(ApplicationStatus)
     private applicationStatusRepository: Repository<ApplicationStatus>,
+    @InjectRepository(ApplicationParcel)
```

Comment:
I do not see references of this repo in the code, do you still need it?

Reply:
Cleaned up, good catch